svc-webhook/src/main/java/com/adnovum/vcms/webhook/service/WebhookService.java

```diff
@@ -53,6 +53,7 @@ public void presentProofEventFromAcaPy(V10PresentationExchange presentationExcha
 		);
 
 		if (presentationExchange.getPresentation() != null) {
```

Comment:
An alternative approach is to use the default terse webhooks. Presentation on the exchange will be null in this case.

We could use the presentation exchange ID and a new call to /present-proof/records/{pres_ex_id}/credentials  to get the revealed attributes. I suggest this as followup ticket to support this flow.